core/serialization/priorities.js

```diff
@@ -6,13 +6,14 @@
 
```

Comment:
It looks like this file was already good and the scripted changes can be reverted?

Reply:
Indentation is supposed to be removed. I have edited though to add back the "The top level namespace for priorities of plugin serializers." to the description.